src/mocks/login.js

```diff
@@ -2,14 +2,14 @@ import { OIDC_AUTHORITY_URL } from "../utils/utils";
 import { user } from "./data";
 
 const SCOPE = process.env.REACT_APP_OIDC_SCOPE;
-const CLIENT_ID = process.env.REACT_APP_OIDC_CLIENT_ID;
+const CLIENT_ID = process.env.REACT_APP// (set this to "NeedsRegistration" to test the full registration and 2FA flow)_OIDC_CLIENT_ID;
```

Comment:
This seems to have broken the mock login?

Reply:
Oops, accidentally checked in some garbage that does not belong there. Fixed it now.